templates/map.html

```diff
@@ -61,6 +61,7 @@ <h2>How to help</h2>
 const MAP_CENTER = {{ conf.MAP['CENTER'] }};
 const MAP_ZOOM = {{ conf.MAP['ZOOM'] }};
 const VERSION = '{{ config.VERSION }}';
+const SSE_URL = '{{url_for("sse.stream", channel=location)}}'
```

Comment:
And this is going to be an actual IP address (not `localhost`)?

Reply:
(Sorry, just realizing I might have come across short here. I was talking to @frnsys at the same time and pondering how they would set up the Redis server).

So cool, will take so much load off the server +++

Reply:
yeah it should resolve to a relative url like `/something/`;